src/typescript-service.ts

```diff
@@ -143,8 +143,13 @@ export class TypeScriptService {
 		this.inMemoryFileSystem = new pm.InMemoryFileSystem(this.root);
 	}
 
-	async shutdown(params = {}, span = new Span()): Promise<void> {
+	async shutdown(params = {}, span = new Span()): Promise<null> {
 		this.projectManager.dispose();
+		return null;
```

Comment:
Good catch. We should only let notification handlers return `void`, and request handlers always `null` if they don't have a result.